text/1001-edgedb-server-control.rst

```diff
@@ -339,12 +348,15 @@ Options
   The name of the EdgeDB instance.  If not specified, the name
   ``default`` is used.
 
-``--version``
+``--to-version``
   Specifies the version of EdgeDB to upgrade to.  If not specified,
   the latest available installed version is used.
 
+``--to-nightly``
+  Specifies that the instance should be upgraded to the nightly version.
+
 ``--nightly``
-  Upgrade to a nightly release.
+  Upgrade all instances currently running nightly.
```

Comment:
Does this upgrade nightlies that cross major versions? I.e. does this upgrade alpha.3 nightly to alpha.4 nightly?

Reply:
Yes. We treat `--nightly` as the latest nightly at all tiimes. I.e. we can't install "alpha.3 nightly".

I've just realized that maybe I'm mistreating nightly. Did you mean that `--version 1-alpha.3 --nightly` would work? Currently, I'm treating `--nightly` and `--version` everywhere as mutually exclusive arguments.

Reply:
Current treatment is OK, but isn't completely clear from the formulation in the RFC.

Reply:
Added a clarification.